packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -519,6 +520,18 @@ export interface CommonProjectProps {
    * @default - no file system locations
    */
   readonly fileSystemLocations?: IFileSystemLocation[];
+
+  /**
+   * A switch controlling whether to add permissions to this project's role
```

Comment:
"A switch controller" is superfluous. It's a boolean toggle, of course it's a switch, and of course it controls something :).

```suggestion
   * Add reporting permissions to this project's role
```

Reply:
Also, divide your comment blocks into a title and a body please, just like git commits and emails.

Reply:
Done.